api/numaresourcesoperator/v1/helper/nodegroup/nodegroup_test.go

```diff
@@ -420,3 +462,13 @@ func findListByNodeGroups(mcps *mcov1.MachineConfigPoolList, nodeGroups []nropv1
 
 	return result, nil
 }
+
+// containedItemsFromSlice returns true if all of b items are part of slice a
+func containsSliceItems(a, b []string) bool {
+	for _, item := range b {
```

Comment:
ok, this can be optimized/cleaned up further later on

Reply:
updated, thanks!